src/app/BoostCalculator.js

```diff
@@ -119,24 +119,26 @@ const BoostCalculator = (props) => {
 
           <label className="flex flex-col">
             <div className="mb-2 dark:text-white text-opacity-80 font-semibold">
-              Artifact Boost Duration Bonus
-              <span className="uppercase bg-gray-100 dark:bg-gray-700 text-xs px-2 py-1 rounded-xl ml-2">
-                Coming Soon
-              </span>
+              Artifact Boost Duration Bonus - &nbsp;
+              <a href="https://wasmegg.netlify.app/artifact-sandbox/#/b/" target="_blank">Calculate @ Artifact Sandbox</a>
             </div>
             <div className="flex space-x-2">
               <div className="flex items-center dark:text-white text-opacity-50">
                 +
               </div>
               <Input
                 type="number"
+                step={0.01}
+                presicion={2}
                 className="flex-grow"
-                placeholder="1, 2, 10, …"
-                value={0}
-                disabled
+                placeholder="1.05, 1.1, 1.5, …"
+                value={dilithiumBoostBonus}
+                onChange={({ target: { value } }) =>
+                  setDilithiumBoostBonus(parseFloat(value, 10) || null)
+                }
               />{' '}
               <div className="flex items-center dark:text-white text-opacity-50">
-                % boost duration
```

Comment:
optional: I think `boost duration` is grammatically correct, but either way sounds good.

Reply:
Haha I had an internal battle with this one - I ultimately settled on "x Boost Duration" at that's the phrasing that the sandbox uses, which is probably how most people will calculate theirs :)